common/debug.h

```diff
@@ -0,0 +1,58 @@
+#include "common/crypto/iss/v1/iss_kerl.h"
+#include "common/helpers/sign.h"
+#include "common/model/bundle.h"
+#include "common/trinary/trit_long.h"
+#include "common/trinary/tryte_long.h"
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "common/crypto/curl-p/digest.h"
+#include "common/model/transaction.h"
+#include "common/trinary/trit_long.h"
+
+#include "logger.h"
+
+/**
+ * Initialize logger
+ */
+void debug_logger_init();
+
+/**
+ * Release logger
+ *
+ * @return
+ * - zero on success
+ * - EXIT_FAILURE on error
+ */
+int debug_logger_release();
+
+/**
+ * Print the content of all the transaction objects in this bundle
+ *
+ * @param bundle[in] Bundle is going to dump message.
+ *
+ */
+void bundle_dump(bundle_transactions_t *bundle);
```

Comment:
Should we have consistent prefix for debugging-oriented functions? That is, something like `dump_` or `ta_dump`.